README.md

```diff
@@ -15,6 +15,7 @@ Material Components for the web is the successor to [Material Design Lite](https
   - [Preact Material Components](https://github.com/prateekbh/preact-material-components)
   - [RMWC: React Material Web Components](https://github.com/jamesmfriedman/rmwc)
   - [Blox Material](https://blox.src.zone/material): Angular Integration Library.
+  - [Vue MDC Adapter](https://github.com/stasson/vue-mdc-adapter): Vue Integration Library.
```

Comment:
Can you add a note about how this uses the Foundation/Adapter approach. This will help clarify the main differences between this Vue library, and matsp's version

Reply:
@lynnjepsen 

I have amended the change with the note.

Reply:
> regarding Theming: 

I guess the guys integrating for angular and react or vue are all asking themselves the same question: what's the best way to allow customization for one component. Should we just delegate to MDC-Web sass or css variables (easiest/cleanest way) or should we also provide a pattern which rely on component properties (kinda complex...) For now the reasonable answer probably is to just rely on MDC-Web sass.

This is a vast subject which may deserve it's whole discussion thread. 


>  Does Vue and Sass play nicely together?

Yes, Kinda. There is indeed a nice feature which comes with vue + webpack that allows you to embed css or sass code in the component itself, and even the ability to scope your styles to the component. This is nice to style or customize a component quickly, but won't work well with a full sass framework like MDC. 

So I decided to not use this specific Vue feature because
- I do not want to rely on webpack specifics (I'm using rollup for UMD distribution, same way most Vue libraries are built)
- It's easier to keep a separate single sass tree to enable theming while avoiding duplicates and rules ordering  issues...

>  supporting theming, and custom colors

I fully rely on MDC-Web sass theming. I have very few customization on top of MDC-Web sass. I only added couple of layouts to help with toolbar/drawer positioning. I follow MDC's color strategy to the letter. 

`dark-theme` is WIP and will be supported as a component property for the components who have a specific dark-them class. I also plan to provide a mdc-dark-theme component wrapper.

For now I do not use css variables, Although it seems really promising and i want to investigate further and see if we can leverage vue reactive system there...